conjure-go-client/httpclient/config.go

```diff
@@ -477,11 +477,7 @@ func newValidatedClientParamsFromConfig(ctx context.Context, config ClientConfig
 		}
 		uris = append(uris, uriStr)
 	}
-	// Plain HTTP clients do not store URIs
-	if !isHTTPClient && len(uris) == 0 {
-		return refreshingclient.ValidatedClientParams{}, werror.ErrorWithContextParams(ctx, "httpclient URLs must not be empty")
```

Comment:
This is the bug fix -- this check was too early. It should be possible to "fix" the empty config with `WithBaseURLs(...)` or a variant, but in that case this would still fail because it operates only on the original ClientConfig struct.